packages/core/src/utils.ts

```diff
@@ -101,14 +104,84 @@ export const checkValidDeployment = async (
       )
     ) {
       throw new Error(
-        `The reference name ${referenceName} inside ${parsedConfig.options.projectName} was already used
-in a previous deployment for this project. You must change this reference name to something other than
-${referenceName} or change the project name to something other than ${parsedConfig.options.projectName}.`
+        `Error: The reference name ${referenceName} inside ${parsedConfig.options.projectName} was already used
+in a previous deployment for this project.
+
+To perform a fresh deployment, you must change this reference name to something other than ${referenceName} or
+change the project name to something other than ${parsedConfig.options.projectName}.
+
+If you wish to upgrade ${parsedConfig.options.projectName} then run to following command:
+npx hardhat chugsplash-upgrade --network ${networkName} ${configPath}`
```

Comment:
I like this!

One nit: I think it's redundant to say `Error: ...` on line 107 since the error message already does automatically. You can see this if you trigger some user-defined error, like calling:
`npx hardhat chugsplash-propose chugsplash/SimpleStorage.config.ts`